hw/top_earlgrey/ip/clkmgr/data/autogen/clkmgr.hjson

```diff
@@ -15,12 +15,12 @@
   scan: "true",
   clocking: [
     {clock: "clk_i", reset: "rst_ni", primary: true},
-    {reset: "rst_main_ni"},
-    {reset: "rst_io_ni"},
-    {reset: "rst_usb_ni"},
-    {reset: "rst_aon_ni"},
-    {reset: "rst_io_div2_ni"},
-    {reset: "rst_io_div4_ni"},
+    {clock: "clk_main_i", reset: "rst_main_ni"},
+    {clock: "clk_io_i", reset: "rst_io_ni"},
+    {clock: "clk_usb_i", reset: "rst_usb_ni"},
+    {clock: "clk_aon_i", reset: "rst_aon_ni"},
+    {clock: "clk_io_div2_i", reset: "rst_io_div2_ni", internal: true},
```

Comment:
QQ: where I can get a hint if the clock is set to `internal` or not?

Reply:
you set it right here actually. It's part of the clock / reset attributes for a particular hjson.
The idea being..the clock is internal..and not supplied by the higher levels, and won't show up in `top_eargrey.hjson` connection list. 

I did this all so that I can use internally derived clocks for clock manager's CDC...